provisioning/ubuntu-provision.sh

```diff
@@ -567,6 +567,14 @@ function install_playwright_dependencies() {
   rm -rf "${temp_dir}"
 }
 
+# Install Launchable with python3
+function install_launchable() {
+  # Install Launchable with python3 in its own virtual environment
+  python3 -m venv launchable
+  launchable/bin/pip --require-virtualenv --no-cache-dir install setuptools wheel
+  launchable/bin/pip --require-virtualenv --no-cache-dir install launchable==${LAUNCHABLE_VERSION}
```

Comment:
```suggestion
  # Install Launchable with python3 in its own virtual environment
  local venv_path=/launchable
  python3 -m venv "${venv_path}"
  "${venv_path}"/bin/pip --require-virtualenv --no-cache-dir install setuptools wheel
  "${venv_path}"/bin/pip --require-virtualenv --no-cache-dir install launchable=="${LAUNCHABLE_VERSION}"
```

WDYT about using a variable to avoid repetition and make it clearer?

Also:
- Use absolute path (recommended to make the function invulnerable to working dir change)
- add double quotes around the `LAUNCHABLE_VERSION` variable (as its content come from the external environment)